rmc/resources/rmc.py

```diff
@@ -497,48 +497,55 @@ def no_breaks_he_path(
 ####################################################################################
 ## MPC related resources
 ####################################################################################
-CURRENT_MPC_PREFIX = f"{MPC_PREFIX}/{CURRENT_GNOMAD_VERSION}/{CURRENT_FREEZE}"
-
-
-def joint_clinvar_gnomad_path() -> str:
-    """
-    Return path to Table containing "population" and "pathogenic" variants.
-
-    Table contains common (AF > 0.001) gnomAD variants ("population") and
-    ClinVar pathogenic/likely pathogenic missense variants in haploinsufficient genes
-    that cause severe disease ("pathogenic") with defined CADD, BLOSUM, Grantham, missense observed/expected ratios,
-    missense badness, and PolyPhen-2 scores.
+joint_clinvar_gnomad = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{MPC_PREFIX}/{CURRENT_GNOMAD_VERSION}/{freeze}/joint_clinvar_gnomad.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing "population" and "pathogenic" variants.
 
-    Table is input to MPC (missense badness, polyphen-2, and constraint) calculations.
+Table contains common (AF > 0.001) gnomAD variants ("population") and
+ClinVar pathogenic/likely pathogenic missense variants in haploinsufficient genes
+that cause severe disease ("pathogenic") with defined CADD, BLOSUM, Grantham, missense observed/expected ratios,
+missense badness, and PolyPhen-2 scores.
 
-    :return: Path to Table.
-    """
-    # TODO: convert back into TableResource
-    return f"{CURRENT_MPC_PREFIX}/joint_clinvar_gnomad.ht"
+Table is input to MPC (missense badness, polyphen-2, and constraint) calculations.
+"""
 
 
-def mpc_model_pkl_path() -> str:
+def mpc_model_pkl_path(freeze: int = CURRENT_FREEZE) -> str:
```

Comment:
do you think we'll generate multiple models? if we only generate MPC on a single freeze, then we'll know from the MPC results path which RMC freeze it's associated with, so we don't need to add this here (but fine keeping this also)

Reply:
We discussed this together and I had added these args to MPC and misbad because they were requested for a couple of functions so I had just applied to everything. So we'll leave it for now